src/text-field-base.css.ts

```diff
@@ -62,33 +110,17 @@ globalStyle(`${fullWidth} > div`, {
 
 export const textArea = style([
     sprinkles({
-        padding: 0,
+        paddingY: 0,
     }),
     {
         resize: 'none',
-        paddingBottom: '8px',
     },
     commonInputStyles,
 ]);
 
-export const textAreaWithLabel = style({
-    // using margin instead of padding to avoid the multiline text being visible through the label
-    marginTop: 28,
-    '@media': {
-        [mq.tabletOrSmaller]: {
-            marginTop: 24,
-        },
-    },
-});
-
-export const textAreaWithoutLabel = style({
-    paddingTop: 16,
-});
-
 export const input = style([
     sprinkles({
         position: 'relative',
-        height: '100%',
```

Comment:
Setting the height of an HTML `<input/>` element is a bad idea. It can mess up the line height and the text alignment (and it behaves in different ways depending on the browser).

More context here: [Link](https://stackoverflow.com/questions/42682530/html-input-field-height-different-in-different-browsers/43219553#43219553)